templates/base/server/routes/auth.py

```
@@ -0,0 +1,136 @@
+    """Request a password reset email. Returns a success message."""
+    await auth_service.request_reset_password(data.email)
+    return {
+        "message": "An email has been sent to your address with a link to reset your password."
+    }
+
+
+@router.get("/reset-password/{token}", status_code=status.HTTP_200_OK)
+async def reset_password(
+    token: str,
+    data: PasswordResetRequest,
+    auth_service: AuthService = Depends(get_auth_service),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)